StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/IntentStatusPollerTest.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  IntentStatusPollerTest.swift
+//  StripePaymentSheetTests
+//
+//  Created by Nick Porter on 9/14/23.
+//
+
+@testable import StripePaymentSheet
+import StripePaymentsTestUtils
+import XCTest
+
+class IntentStatusPollerTest: XCTestCase {
+    var sut: IntentStatusPoller!
+    var mockIntentRetriever: MockPaymentIntentRetriever!
+    var mockDelegate: MockIntentStatusPollerDelegate!
+    var intentRetrieverExpectation: XCTestExpectation!
+    var delegateExpectation: XCTestExpectation!
+
+    override func setUp() {
+        super.setUp()
+        mockIntentRetriever = MockPaymentIntentRetriever()
+        mockDelegate = MockIntentStatusPollerDelegate()
+        sut = IntentStatusPoller(retryInterval: 1.0, intentRetriever: mockIntentRetriever, clientSecret: "test_client_secret")
```

Comment:
nit: Might as well use a smaller interval for this test?  Like 0.1

Reply:
Reduced to 0.1